project-base/app/config/packages/sonata_block.yaml

```diff
@@ -0,0 +1,2 @@
+sonata_block:
+    http_cache: false
```

Comment:
can we unify all sonata configurations to `sonata_admin.yaml` file?
Now we have sonata_block definition here and in `sonata_admin.yaml`. 

Same applies also to `sonata_form.yaml`